pyramid_oereb/contrib/interlis/import_theme_transfer_structure.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this should not be the way how to offer this. I would like more an approach of bash script or even a Makefile to avoid subprocess calls out of python.